### PR TITLE
feat(files): peace_and_quiet/mobs

### PR DESCRIPTION
**Pull request by SquatchHunter**
fixes: #234
resolves: #232
resolves: #231

* add missing packs (no ender dragon death sound & no wither spawn sound)
* correct pack_icons
* add fullstop to all descriptions in packs.json

By checking the following boxes with an X, you ensure that:

- [x] The pack was tested ingame in at least one device.
- [x] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [x] The pack code follows the style guide.
- [x] The commits follow the contribution guidelines.
- [x] The PR follows the contribution guidelines.

- [x] (Optional) Tested in Windows
- [ ] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS

### CODE REVIEW

**Comment by PAPISOP** [comment]
Descriptions aren't updated for new additions to the packs
like quieter villagers description should be "Decreases Villager, Wandering Trader, and workstation sounds by 95%." (this is an example, there are many 2 more packs description to update) @SquatchHunter

**Comment by PAPISOP** [comment]
i have also noticed that some of the files are not formatted

**Comment by drav0011** [comment]
Full stops were removed for stylistic reasons
Also revise resource_packs/files/peace_and_quiet/mobs/quieter_shulkers/sounds.json formatting

**Comment by PAPISOP** [comment]
Descriptions of packs which got new additions aren't updated (I am not talking about comma issues with description, I am talking about missing stuff in descriptions such as in quieter bees, its not mentioned that beehives and bee nests are quieter which it is in vanilla tweaks so just see vanilla tweaks and copy paste their descriptiops) @SquatchHunter

File: resource_packs/packs.json [hunk 1]
```diff
@@ -2274,6 +2288,11 @@
 			"id": "parity",
 			"name": "Parity",
 			"packs": [
+				{
+					"id": "java_menu_title",
+					"name": "Java Menu Title",
+					"description": "Gives the title on the main menu screen its look from Minecraft: Java Edition"
+				},
```
Comment (PAPISOP) [inline resource_packs/packs.json:2295]: This needs to be removed as this pack was removed in this pr #220

File: resource_packs/packs.json
Comment (PAPISOP) [inline resource_packs/packs.json:1]:
```
{
					"id": "quieter_water",
					"name": "Quieter Water",
					"description": "Decreases Water, swimming, water drops from Dripstones and underwater sound events by 95%."
},
```
Update quieter water's description

**Review by PAPISOP — CHANGES_REQUESTED** [review]
other than that lgtm

**Comment by PAPISOP** [comment]
other than that lgtm

File: resource_packs/packs.json [hunk 2]
```diff
@@ -939,7 +953,7 @@
 				{
 					"id": "music_disc_redstone_level_preview",
 					"name": "Music Disc Redstone Level Preview",
-					"description": "Gives all Music Discs a redstone power level indicator in the tooltip"
+					"description": "Displays the redstone power level for each music disk in the tooltip"
```
Comment (PAPISOP) [inline resource_packs/packs.json:956]: typo in "disk", should be "disc"

File: resource_packs/packs.json [hunk 3]
```diff
@@ -1848,7 +1862,7 @@
 				{
 					"id": "pre_1_18_stripped_dark_oak",
 					"name": "Pre-1.18 Stripped Dark Oak",
-					"description": "Reverts stripped dark oak to use its pre-1.18 color"
+					"description": "Reverts stripped dark oak to use its pre-1.18 color."
```
Comment (PAPISOP) [inline resource_packs/packs.json:1865]: full stop needs to be removed

**Review by PAPISOP — CHANGES_REQUESTED** [review]

File: resource_packs/packs.json
Comment (SquatchHunter) [reply resource_packs/packs.json:1]: I have.

File: resource_packs/packs.json (hunk 1 above)
Comment (SquatchHunter) [reply resource_packs/packs.json:2295]: already removed, you are looking at old commits.

File: resource_packs/packs.json
Comment (PAPISOP) [reply resource_packs/packs.json:1]: > I have. that is 1 to 1 what VT has

no? its not 1 to 1 to vt
image

**Review by drav0011 — APPROVED** [review]
lgtm

**Pull request merged**